data/stylesheets/asciidoctor-default.css

```diff
@@ -361,8 +361,13 @@ div.unbreakable{page-break-inside:avoid}
 .white-background{background-color:#fafafa}
 .yellow{color:#bfbf00}
 .yellow-background{background-color:#fafa00}
-span.icon>.fa{cursor:default}
-a span.icon>.fa{cursor:inherit}
+svg{
```

Comment:
Changes to the stylesheet need to be made in the stylesheet factory, then rebuilt as a minimized file. See https://github.com/asciidoctor/asciidoctor-stylesheet-factory